tests/org.eclipse.n4js.hlc.tests/src/org/eclipse/n4js/hlc/tests/InstallCompileRunN4jscExternalImportsTest.java

```diff
@@ -48,22 +54,22 @@ public void deleteWorkspace() throws IOException {
 	 */
 	@Test
 	@Ignore // remove @Ignore when GH-887 is merged
-	public void testCompileAndRunWithExternalDependencies() throws IOException, ExitCodeException {
+	// continue here
+	public void testCompileAndRunWithExternalDependencies() {
 		final String wsRoot = workspace.getAbsolutePath().toString();
 		final String packages = wsRoot + "/packages";
-		final String fileToRun = packages + "/external.project/src/Main.n4js";
+		final String fileToRun = packages + "/external.project/src-gen/Main.js";
+
+		N4jscOptions options = COMPILE(workspace);
+		CliResult cliResult = main(options);
+		assertEquals(cliResult.toString(), 69, cliResult.getTranspiledFilesCount());
+
+		String expectedString = "react is not undefined true\n"
+				+ "react-dom is not undefined true\n"
+				+ "imports from libs are different true";
 
-		final String[] args = {
-				"--installMissingDependencies",
-				"--runWith", "nodejs",
-				"--run", fileToRun,
-				"--projectlocations", packages,
-				"--buildType", BuildType.allprojects.toString()
-		};
```

Comment:
Where did all these command line arguments go?

Reply:
`--run` - removed, since the new CLI does not support execution of node.js
`--projectlocations` - removed, since the new CLI searches all projects like the lsp does
`--buildType` - removed, since the new CLI has only one build type which is _build all projects found at the given location_.